springwolf-core/src/main/java/io/github/stavshamir/springwolf/asyncapi/scanners/channels/operationdata/AbstractOperationDataScanner.java

```diff
@@ -1,9 +1,9 @@
 package io.github.stavshamir.springwolf.asyncapi.scanners.channels.operationdata;
 
-import com.asyncapi.v2.binding.channel.ChannelBinding;
-import com.asyncapi.v2.binding.operation.OperationBinding;
 import com.asyncapi.v2._0_0.model.channel.ChannelItem;
 import com.asyncapi.v2._0_0.model.channel.operation.Operation;
```

Comment:
I think that we should also upgrade all the `com.asyncapi.v2._0_0` imports to `com.asyncapi.v2._6_0`. Those new models are aligned with the different expectations of the new spec

Reply:
True, let me change the PR to draft for now.